### PR TITLE
EZP-26003: Add search button in PlatformUI

### PR DESCRIPTION
**Pull request by StephaneDiot**
JIRA: https://jira.ez.no/browse/EZP-26003

## Description

First part of the search epic : this only add the search button in the discoverybar and the route to the app

## Tests

unit and manually tested

### CODE REVIEW

**Comment by StephaneDiot** [comment]
ping @dpobel @yannickroger

File: Resources/public/js/apps/ez-platformuiapp.js [hunk 1]
```diff
@@ -903,6 +903,12 @@ YUI.add('ez-platformuiapp', function (Y) {
                     service: Y.eZ.ServerSideViewService,
                     view: "serverSideView",
                     callbacks: ['open', 'checkUser', 'handleSideViews', 'handleMainView']
+                }, {
+                    name: "viewSearch",
```
Comment (dpobel) [inline Resources/public/js/apps/ez-platformuiapp.js]: should be moved a bit above keep the routes *sorted*

**Comment by dpobel** [comment]
besides the inline comment +1

**Comment by yannickroger** [comment]
+1

**Pull request merged**